tools/StaticAnalysis/Exceptions/Az.Synapse/SignatureIssues.csv

```diff
@@ -1,4 +1,5 @@
 "AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
-"D:\code\AzureSDK\azure-powershell\artifacts\Debug\Az.Synapse\Microsoft.Azure.PowerShell.Cmdlets.Synapse.dll","Microsoft.Azure.Commands.Synapse.SubmitAzureSynapseSparkJob","Submit-AzSynapseSparkJob","1","8100","Submit-AzSynapseSparkJob Does not support ShouldProcess but the cmdlet verb Submit indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
-"D:\code\AzureSDK\azure-powershell\artifacts\Debug\Az.Synapse\Microsoft.Azure.PowerShell.Cmdlets.Synapse.dll","Microsoft.Azure.Commands.Synapse.StartAzureSynapseSparkSession","Start-AzSynapseSparkSession","1","8100","Start-AzSynapseSparkSession Does not support ShouldProcess but the cmdlet verb Start indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
-"D:\code\AzureSDK\azure-powershell\artifacts\Debug\Az.Synapse\Microsoft.Azure.PowerShell.Cmdlets.Synapse.dll","Microsoft.Azure.Commands.Synapse.InvokeAzureSynapseSparkStatement","Invoke-AzSynapseSparkStatement","1","8100","Invoke-AzSynapseSparkStatement Does not support ShouldProcess but the cmdlet verb Invoke indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Microsoft.Azure.PowerShell.Cmdlets.Synapse.dll","Microsoft.Azure.Commands.Synapse.SubmitAzureSynapseSparkJob","Submit-AzSynapseSparkJob","1","8100","Submit-AzSynapseSparkJob Does not support ShouldProcess but the cmdlet verb Submit indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Microsoft.Azure.PowerShell.Cmdlets.Synapse.dll","Microsoft.Azure.Commands.Synapse.StartAzureSynapseSparkSession","Start-AzSynapseSparkSession","1","8100","Start-AzSynapseSparkSession Does not support ShouldProcess but the cmdlet verb Start indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Microsoft.Azure.PowerShell.Cmdlets.Synapse.dll","Microsoft.Azure.Commands.Synapse.InvokeAzureSynapseSparkStatement","Invoke-AzSynapseSparkStatement","1","8100","Invoke-AzSynapseSparkStatement Does not support ShouldProcess but the cmdlet verb Invoke indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
```

Comment:
Please take the suggested action to support shouldprocess for these 3 cmdlets

Reply:
@isra-fel These three top issues are existing issues.  Can we just suppress these static analysis results because I think the current implementation (without ShouldProcess) are better in user experience?

Reply:
Supporting shouldprocess means adding two *optional* parameters `-WhatIf` and `-Confirm`. They do not affect default behavior of the cmdlet, but only provide additional information or confirmation when user is uncertain about it. So it is definitely better to support.

Reply:
I see. Thanks for your suggestions. I will fix this in a separate PR.

Reply:
@isra-fel I created a PR to fix this issue: https://github.com/Azure/azure-powershell/pull/12670 . Could you please take a look when you have some time? Thanks in advance.

Reply:
Thanks! I have approved #12670 and will merge it by today. @wonner please rebase master after it's merged.